blog/models.py

```diff
@@ -81,6 +82,11 @@ def image(self):
         if len(self.images):
             return self.images[0]
 
+    def delete(self, *args, **kwargs):
+        super(Article, self).delete(*args, **kwargs)
```

Comment:
When overriding a method and still calling its parent, we should always at least pass the same parameters (that you now do) but also return what the parent returns. It may be `None` today but a boolean, a count or other values in the future parent method updates.
Example of how to achieve this:
```
parent = super(…
…
return parent
```